catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -86,6 +102,18 @@ define(['application',
                     }
                 );
 
+                var that = this;
+                viewer.camera.moveEnd.addEventListener(function() {
+                    if(that.clustering) {
+                        var cartographic = new Cesium.Cartographic();
+                        var metersToKm = 0.001;
+                        viewer.scene.mapProjection.ellipsoid.cartesianToCartographic(viewer.camera.positionWC, cartographic);
+                        var cameraHeight = (cartographic.height * metersToKm).toFixed(1);
```

Comment:
There's a check in `MapClustering.js` that won't cluster if the camera is below a certain height (default 10m).  I haven't observed any issues while zooming